model-optimizer/extensions/ops/interpolate.py

```diff
@@ -23,6 +28,24 @@ class Interpolate(Op):
     enabled = False
 
     def __init__(self, graph: Graph, attrs: dict):
+        self.infers = {
+            'opset1': Interpolate.infer_for_opset1,
+            'opset4': Interpolate.infer_for_opset4
+        }
+
+        self.attributes_for_opsets = {
+            'opset1': [
+                ('axes', lambda node: ','.join(map(str, node.axes))),
+                'mode', 'align_corners', 'antialias', 'pads_begin', 'pads_end',
+            ],
+            'opset4': [
+                'mode', 'antialias', 'nearest_mode', 'cube_coeff', 'coordinate_transformation_mode',
+                'shape_calculation_mode',
+                ('pads_begin', lambda node: pad_attribute_to_str(node, 'pads_begin')),
```

Comment:
Why there is a difference between `pads_begin` and `pads_end` in Interpolate-1 and Interpolate-4?  Why in one case we just write `'pads_begin'` while in another case we write `('pads_begin', lambda node: pad_attribute_to_str(node, 'pads_begin'))`?

Reply:
Because, according to the specification of Interpolate-1, `pads_begin` and `pads_end` have type `int`, but, according to the specification of Interpolate-4, `pads_begin` and `pads_end` have type `int[]`.